rhino/src/main/java/org/mozilla/javascript/ScriptRuntime.java

```diff
@@ -5828,6 +5828,21 @@ public static Scriptable wrapRegExp(Context cx, Scriptable scope, Object compile
 
     public static Scriptable getTemplateLiteralCallSite(
             Context cx, Scriptable scope, Object[] strings, int index) {
+        TemplateLiteralProxy proxy = cx.getTemplateLiteralProxy();
+        if (proxy != null) {
+            return proxy.getCallSite(cx, scope, strings, index);
```

Comment:
Can you explain to me what exactly is supposed to happen here? As far as I know, the same code is simply being called in a somewhat cumbersome way. The intention is that the JIT can then compile it more optimally?